app/controllers/maps_controller.rb

```diff
@@ -133,6 +152,48 @@ def events
     end
   end
 
+  # GET maps/:id/approve_access/:request_id
+  def approve_access
+    request = AccessRequest.find_by_id(params[:request_id])
```

Comment:
this can just be AccessRequest.find(params[:request_id]), the find_by helpers are deprecated

You can also do AccessRequest.find_by(id: ...) or AccessRequest.find_by(name: ...), etc, which is why they took the named helpers out